docs/datatypes/io.md

```diff
@@ -164,38 +164,30 @@ val composite = action.ensuring(cleanupAction)
 ### A full working example on using brackets
 ```scala mdoc:silent
 
-import scalaz.zio.{ App, DefaultRuntime, Task, UIO }
+import scalaz.zio.{App, Task, UIO}
 
-import java.io.{ File, FileInputStream }
+import java.io.{File, FileInputStream }
 import java.nio.charset.StandardCharsets
 
 object Main extends App {
 
-  val rt = new DefaultRuntime {}
-
   // run my bracket
   def run(args: List[String]) =
-    mybracket.orDie.map(_ => 0)
+    mybracket.orDie.const(-1) // return -1 on fail
 
   def closeStream(is: FileInputStream) =
-    UIO.effectTotal(is.close())
+    UIO(is.close())
 
   def convertBytes(is: FileInputStream) =
-    Task.effect(println(new String(is.readAllBytes(), StandardCharsets.UTF_8)))
+    Task.effect(println(new String(is.readAllBytes(), StandardCharsets.UTF_8))) // Java 11+
 
   val mybracket: UIO[Unit] = {
-
     UIO {
-
       val file = new File("/tmp/hello")
-
       val string = Task.effect(new FileInputStream(file)).bracket(closeStream)(convertBytes).orDie
-
       // Print effect
-      rt.unsafeRun(string)
-
+      unsafeRun(string)
```

Comment:
We don't encourage using `unsafeRun` directly, and it's not needed here as the `App` will run the effect for you.

Reply:
e.g.
```scala
for {
  file <- Task(new File("/tmp/hello"))
  string <- Task(new FileInputStream(file)).bracket(closeStream)(convertBytes)
} yield string
```